rdrf/rdrf/settings.py

```diff
@@ -317,6 +319,8 @@
 CSRF_HEADER_NAME = env.get("csrf_header_name", 'HTTP_X_CSRFTOKEN')
 CSRF_TRUSTED_ORIGINS = env.getlist("csrf_trusted_origins", ['localhost'])
 
+# Content Security Policy
+CSP_DEFAULT_SRC = tuple(filter(None, ["'self'", "'unsafe-inline'", get_static_url_domain(env.get("STATIC_URL", ""))]))
```

Comment:
Shouldn't we also set: `CSP_OBJECT_SRC = ('none',) ` based on: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/object-src ?

Reply:
Actually this might conflict with Angelman where we have the consent pdf displayed in an iframe

Reply:
I think you can set ``CSP_OBJECT_SRC`` to ``None`` as the iframes use ``CSP_FRAME_SRC``.
We will have to accept * from iframes on the consent page though...